src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,98 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final float LOAD_FACTOR = 0.75f;
+    private static final int INCREASE = 2;
+    private int currentCapacity;
```

Comment:
```suggestion
```
In your code you use table.lenth, in this case this variable is redundant

Reply:
i use this variable as CURRENT capacity, it is used in increaseCapacity method, not sure if it is redundant in this case

Reply:
let's remove this field
we don't actually need it
because we can call `table.length` in code where we need capacity